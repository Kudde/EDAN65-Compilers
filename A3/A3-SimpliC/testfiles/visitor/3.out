Maximum statement nesting: 3
