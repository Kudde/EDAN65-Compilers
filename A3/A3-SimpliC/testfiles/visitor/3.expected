Maximum statement nesting: 5
